src/labels/CustomLabels.labels

```diff
@@ -5128,6 +5290,22 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>Field Label</shortDescription>
         <value>Field Label</value>
     </labels>
+    <labels>
+        <fullName>commonFieldsNotFound</fullName>
```

Comment:
@dfuller114 I realize your story is still in flux, a bit, but these labels are related to the "broken reference in template builder" story I beliecve